lib/config.js

```diff
@@ -8,6 +8,16 @@ require('bedrock-validation');
 
 const cfg = config['authn-token-http'] = {};
 
+cfg.cookies = {};
+// client id to be baked into cookies that is used in client registration
+cfg.cookies.clientId = {
+  options: {
+    name: 'cid',
+    // 30 days expiration
+    maxAge: 30 * 24 * 60 * 60 * 1000
+  }
```

Comment:
Sorry, I was unclear about what to do here. We want the name to be separate from the options ... so we can just pass the options directly into the `res.cookie` API below.

```suggestion
  name: 'cid',
  options: {
    // 30 days expiration
    maxAge: 30 * 24 * 60 * 60 * 1000,
    httpOnly: true
  }
```